matching-notification-api/src/main/java/org/phenotips/matchingnotification/finder/MatchFinder.java

```diff
@@ -43,4 +43,13 @@
      */
     List<PatientMatch> findMatches(Patient patient);
 
+    /**
+     * Record start time for running matches search.
+     */
+    void recordStartMatchesSearch();
```

Comment:
I would name this method a bit different to at least differentiate it from the same method in `MatchFinderManager.java`

E.g. this can be named `RecordMatcherStart`, and the other `RecordAllMatchesStart` or smth like that , to visually differentiate them and what they do